pkgs/misc/screensavers/betterlockscreen/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  stdenv, makeWrapper, fetchFromGitHub, substituteAll,
+  imagemagick, i3lock-color, xdpyinfo, xrandr, bc, feh
+}:
+
+stdenv.mkDerivation rec {
+  name = "betterlockscreen-${version}";
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "pavanjadhaw";
+    repo = "betterlockscreen";
+    rev = version;
+    sha256 = "0jc8ifb69shmd0avx6vny4m1w5dfxkkf5vnm7qcrmc8yflb0s3z6";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  patches = [ (substituteAll {
```

Comment:
The substituteAll shouldn't be needed

Reply:
So replace with `substitute` instead, as it is only one occurrence, or?
I have only ever seen with `substituteAll`, including example here - https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/kitty/default.nix

Reply:
Just do: `patches = [ ./replace-i3lock.patch ];`